src/content/docs/en/guides/deploy/google-cloud.mdx

```diff
@@ -44,7 +44,7 @@ Once the Dockerfile is created, build it into an image and push it to Google Clo
 
 **Build locally using Docker**:
 
-Use the `docker build` command to build the image, `docker tag` to give it a tag, then `docker push` to push it to a registry. In the case of Google Cloud, [`Google Cloud Registry`](https://cloud.google.com/container-registry/docs/pushing-and-pulling#add-registry) is the easiest option, but you can also use [Docker Hub](https://hub.docker.com/).
```

Comment:
The original link directly points to `#add-registry` but it was skipping a few prerequisites before the section, so I think it should navigate to the beginning of page.